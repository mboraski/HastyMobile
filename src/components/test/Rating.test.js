// import 'react-native';
// import React from 'react';
// import { shallow } from 'enzyme';
//
// import Rating from '../Rating';
//
// describe('Rating', () => {
//     it('renders correctly', () => {
//         const render = shallow(<Rating />);
//         expect(render).toMatchSnapshot();
//     });
//     it('handles onChange', () => {
//         const onChange = jest.fn();
//         const render = shallow(<Rating onChange={onChange} />);
//         render.instance().handleRating(3);
//         expect(onChange.mock.calls[0][0]).toEqual(3);
//     });
// });
