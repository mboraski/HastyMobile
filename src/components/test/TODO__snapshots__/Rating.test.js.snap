// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexDirection": "row",
      },
      undefined,
    ]
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    key="0"
    onPress={[Function]}
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "marginRight": 10,
          },
          Object {
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    />
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    key="1"
    onPress={[Function]}
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "marginRight": 10,
          },
          Object {
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    />
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    key="2"
    onPress={[Function]}
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "marginRight": 10,
          },
          Object {
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    />
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    key="3"
    onPress={[Function]}
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "marginRight": 10,
          },
          Object {
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    />
  </TouchableOpacity>
  <TouchableOpacity
    activeOpacity={0.2}
    key="4"
    onPress={[Function]}
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "marginRight": 10,
          },
          Object {
            "height": 30,
            "width": 30,
          },
          undefined,
        ]
      }
    />
  </TouchableOpacity>
</View>
`;
